classes/publication/PKPPublication.php

```diff
@@ -466,6 +468,35 @@ public function getLanguages(?array ...$additionalLocales): array
             ->values()
             ->toArray();
     }
+
+    /**
+     * Get the current version data
+     */
+    public function getCurrentVersionData(): ?VersionData 
```

Comment:
I think `Current` here is confusing... Maybe remove it?

Reply:
Changed!